release-notes/aws/v11.2.0.md

```diff
@@ -1,8 +1,16 @@
-# :zap: Giant Swarm Release 11.1.1 for AWS is now active for you! :zap:
```

Comment:
Why does the title completely break convention now?

Reply:
Changed